examples/nirfmxbluetooth/txp-basic.py

```diff
@@ -296,19 +296,22 @@ def raise_if_error(response):
     print(
         f"Peak to Average Power Ratio Maximum (dB)        : {peak_to_average_power_ratio_maximum}"
     )
-    print(f"EDR GFSK Average Power Mean (dBm)               : {edrgfsk_average_power_mean}")
-    print(f"EDR DPSK Average Power Mean (dBm)               : {edrdpsk_average_power_mean}")
-    print(
-        f"EDR DPSK GFSK Average Power Ratio Mean (dB)     : {edR__dpsK__gfsk_average_power_ratio_mean}"
-    )
 
-    print("------------------LE CTE Reference Period Measurement------------------")
-    print(
-        f"Average Power Mean (dBm)                                         : {reference_period_average_power_mean}"
-    )
-    print(
-        f"Peak Absolute Power Deviation Maximum (%)                         : {reference_period_peak_absolute_power_deviation_maximum}\n"
-    )
+    # Uncomment this block if these values are generated by your measurement:
```

Comment:
Personally, I don't think it needs to mirror the C examples exactly -- or if so, then maybe the C examples should be updated! But I would prefer that rather than commenting out parts of the examples, we have a configurable variable that we use to automatically skip those prints, based on what we expect for those inputs. For example, see the `auto_level` variable in this txp-basic example. It's not something we pass down to the grpc calls; it's only used to decide which code path to take in the example. And if changing `auto_level` to false would necessarily result in some of the outputs being NaN, then we could put those corresponding prints under an `if auto_level is False` check.

Reply:
The quote from Dharti is:

> When ACP Offset Channel Mode Property is set to ‘Symmetric’, Lower and Upper margin results returns NaN value that is because mask is not applicable for ‘Symmetric’ mode. For Basic Rate packets, results such as EDR GFSK Average Power Mean, EDR DPSK Average Power Mean, EDR DPSK GFSK Average Power Ratio Mean, Average Power Mean, and Peak Absolute Power Deviation Maximum are not valid, Hence NaN is returned in these results. 

So would the condition in this case be:

```python
if offset_channel_mode != nirfmxbluetooth_types.ACP_OFFSET_CHANNEL_MODE_SYMMETRIC:
```

Or whatever the channel mode is where those are applicable?

I think that would be useful doing that like David's suggestion would be useful.

I also think it could be useful if there was a comment like "Lower and Upper margin results are not applicable for ACP_OFFSET_CHANNEL_MODE_SYMMETRIC" in this place. I don't think the current comment is useful; nor the commented out print statements. We've already shown them how to read the fields from the response. And they can figure out how to print them. If we're going to add anything here it's "why are we not printing them?" not "how do I print them if I want to?".

Reply:
I've updated the code to incorporate the feedback in this thread.

Reply:
Feedback looks a lot better I am glad we could remove the commented out section.

Reply:
The changes look good. Thanks for following up & fixing this.
I agree that we need not have to exactly mirror the C examples & we can make the user experience better with gRPC examples within limits.